action.yaml

```diff
@@ -0,0 +1,51 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: SkyWalking Infra E2E
+description: End-to-End Tesing framework that help to set up, verify E2E tests.
+author: Apache SkyWalking
+inputs:
+  e2e-file:
+    description: File path of e2e file
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+        mkdir -p skywalking-e2e/bin
```

Comment:
Let's use an absolute path, for example, `/home/e2e`

Reply:
> Let's use an absolute path, for example, `/home/e2e`

Otherwise, it may pollute the users workspace

Reply:
please use `/home/skywalking-e2e`

Reply:
I have tried using an absolute path on this, but the GHA composite mode is not supported. please see: https://github.com/mrproliu/skywalking/pull/1/checks?check_run_id=2869437957